### PR TITLE
[OSDOCS-5012]: Full Azure CPMS support (rel notes)

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.13

Issue:
[OCPCLOUD-1764](https://issues.redhat.com//browse/OCPCLOUD-1764) | [OSDOCS-5012](https://issues.redhat.com//browse/OSDOCS-5012)

Link to docs preview:
[Additional platform support for control plane machine sets](https://55409--docspreview.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-13-release-notes.html#ocp-4-13-mapi-cpms-platform-support)

QE review:
- [x] QE has approved this change.

Additional information:
* Main docs: #55407

### CODE REVIEW

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -94,6 +94,17 @@ With this enhancement, users can set the `--import-mode` flag to `Legacy` or `Pr
 [id="ocp-4-13-machine-api"]
 === Machine API
 
+[id="ocp-4-13-mapi-cpms-platform-support"]
+==== Additional platform support for control plane machine sets
```
Comment (jeana-redhat) [inline release_notes/ocp-4-13-release-notes.adoc:98]: This says additional platform support because I plan to stick GCP under here as well.

**Comment by jeana-redhat** [comment]
@odvarkadaniel this is also ready for review, PTAL

File: release_notes/ocp-4-13-release-notes.adoc [hunk 2]
```diff
@@ -94,6 +94,17 @@ With this enhancement, users can set the `--import-mode` flag to `Legacy` or `Pr
...
+
+This release includes an enhancement to the user experience for the control plane machine set on Microsoft Azure clusters.
+
+For Azure clusters that are installed with {product-title} version 4.13, you are no longer required to create a control plane machine set custom resource (CR). Instead, an inactive CR is generated for the cluster and can be activated after you verify that the values in the CR are correct for your control plane machines.
+
+Azure clusters that are upgraded to version 4.13 from an earlier version still require xref:../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-creating-cr_cpmso-getting-started[manual CR creation].
```
[the remark this answers is not in the document]
Comment (jeana-redhat) [reply release_notes/ocp-4-13-release-notes.adoc:104]: This is the same issue as in the other Azure PR, will fix. Thanks for explaining!

File: release_notes/ocp-4-13-release-notes.adoc [hunk 3]
```diff
@@ -104,9 +104,7 @@ With {product-title} {product-version}, running `oc describe` on an image now re
 
 This release includes an enhancement to the user experience for the control plane machine set on Microsoft Azure clusters.
 
-For Azure clusters that are installed with {product-title} version 4.13, you are no longer required to create a control plane machine set custom resource (CR). Instead, an inactive CR is generated for the cluster and can be activated after you verify that the values in the CR are correct for your control plane machines.
-
-Azure clusters that are upgraded to version 4.13 from an earlier version still require xref:../machine_management/control_plane_machine_management/cpmso-getting-started.adoc#cpmso-creating-cr_cpmso-getting-started[manual CR creation].
+For Azure clusters that are installed with or upgraded to {product-title} version 4.13, you are no longer required to create a control plane machine set custom resource (CR). Instead, an inactive CR is generated for the cluster and can be activated after you verify that the values in the CR are correct for your control plane machines.
```
Comment (jeana-redhat) [inline release_notes/ocp-4-13-release-notes.adoc:107]: I think this fixes it, if I am understanding correctly now. @odvarkadaniel PTAL :)

Comment (odvarkadaniel) [reply release_notes/ocp-4-13-release-notes.adoc:107]: Nice, LGTM!

**Comment by jeana-redhat** [comment]
@sunzhaohua2 PTAL at this as well :)

File: release_notes/ocp-4-13-release-notes.adoc [hunk 4]
```diff
@@ -99,6 +99,15 @@ With {product-title} {product-version}, running `oc describe` on an image now re
 [id="ocp-4-13-machine-api"]
 === Machine API
 
+[id="ocp-4-13-mapi-cpms-platform-support"]
+==== Additional platform support for control plane machine sets
+
+This release includes an enhancement to the user experience for the control plane machine set on Microsoft Azure clusters.
+
+For Azure clusters that are installed with or upgraded to {product-title} version 4.13, you are no longer required to create a control plane machine set custom resource (CR). Instead, an inactive CR is generated for the cluster and can be activated after you verify that the values in the CR are correct for your control plane machines.
```
Comment (sunzhaohua2) [inline release_notes/ocp-4-13-release-notes.adoc:107]: @jeana-redhat Not sure if I understand correctly, but for fresh install, CPMS is in Active state. If delete or upgrade to 4.13, CPMS is InActive.

Comment (odvarkadaniel) [reply release_notes/ocp-4-13-release-notes.adoc:107]: Ah yes. That is true. Sorry for confusion @jeana-redhat.

Comment (jeana-redhat) [reply release_notes/ocp-4-13-release-notes.adoc:107]: Ok, so I think that means that in #55407, the table should actually look very similar for AWS and Azure (just different versions). I will fix this PR and update the other to match, then tag for validation again :)

Comment (jeana-redhat) [reply release_notes/ocp-4-13-release-notes.adoc]: Updated this PR and the other one. @odvarkadaniel and @sunzhaohua2 PTAL :)

Comment (sunzhaohua2) [reply release_notes/ocp-4-13-release-notes.adoc]: Thanks @jeana-redhat 
LGTM

File: release_notes/ocp-4-13-release-notes.adoc [hunk 5]
```diff
@@ -157,6 +157,17 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
...
+[id="ocp-4-13-mapi-cpms-platform-support"]
+==== Additional platform support for control plane machine sets
+
+This release includes an enhancement to the user experience for the control plane machine set on Microsoft Azure clusters. For Azure clusters that are installed with or upgraded to {product-title} version 4.13, you are no longer required to create a control plane machine set custom resource (CR).
+
+* Clusters that are installed with version 4.13 have a control plane machine set that is active by default.
+
+* For clusters that are upgraded to version 4.13, an inactive CR is generated for the cluster and can be activated after you verify that the values in the CR are correct for your control plane machines.
+
+For more information, see xref:../machine_management/control_plane_machine_management/cpmso-getting-started.adoc[Getting started with the Control Plane Machine Set Operator].
```
Comment (adellape) [inline release_notes/ocp-4-13-release-notes.adoc:169]: Anchor after `.adoc`

**Pull request merged**